ui/ducks/send/swap-and-send-utils.ts

```diff
@@ -140,6 +140,9 @@ const QUOTE_VALIDATORS = [
     type: 'number',
   },
 ];
+
+const SWAP_AND_SEND_SLIPPAGE = '2';
```

Comment:
just confirming, does swaps have a default slippage and is it the same as this?

Reply:
I couldn't find one. I just found this code in `review-quote.js`:

```
const [slippageErrorKey] = useState(() => {
    const slippage = Number(fetchParams?.slippage);
    if (slippage > 0 && slippage <= 1) {
      return SLIPPAGE_LOW_ERROR;
    } else if (slippage >= 5 && slippage <= MAX_ALLOWED_SLIPPAGE) {
      return SLIPPAGE_HIGH_ERROR;
    }
    return '';
  });
```

This indicates that the range for swaps quotes is 1-5 (both exclusive). I'd imagine we'd want to be on the lower end given the typical use cases for a send, so 2 seems like a clean number to choose.